prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+   const fullName = 'Sophia Lee';
+   console.log(fullName);
+   const numberOfPets = 2;
+   console.log(numberOfPets);
+   const likesVegetables = 'true';
+   console.log(likesVegetables);
+   const nothing = 'null';
+   console.log(nothing);
```

Comment:
Hi Sophia, be sure to leave the quotes off of your boolean and null, so that the JavaScript engine does not interpret them as a string.